visualize-admin/visualization-tool
# fix: shared yAdjustments to Ruler and Dots

**This PR**
- This PR resolves the misalignment of the Ruler and HoverDots that indicate the values
- This PR resolves # 110 of the private visualize 

---

**Description**
@bprusinowski I'm not particularly happy w/ this solution, because it's so edge case focused. However let me know what you think the primary reason for this issue is the `app/charts/combo/combo-line-container.tsx`

---

### How to test
1. Go to this [link](https://github.com/visualize-admin/visualization-tool-private/issues/110) 
2. See how the HoverDots and Ruler are misaligned ❌
3. Go to this [link](https://vercel.live/open-feedback/visualization-tool-git-fix-dual-line-chart-hover-ixt1.vercel.app?via=pr-comment-visit-preview-link&passThrough=1) 
4. Create a dual Line Chart
5. Publish the Chart 
6. Decrease the Screen size until the axis titles are atleast 2 lines
7. See how the HoverDots and Ruler are no longer misaligned ✅
